tools/pubsub2inbox/examples/cai-example.yml

```diff
@@ -0,0 +1,63 @@
+#   Copyright 2021 Google LLC
```

Comment:
I think this file should be named `cai-example.yaml` for consistency (note the missing `a` in the extension)